### DISCUSSION TITLE
Pdf file page formating

### DISCUSSION THREAD

**Discussion by cedounet** [hendricius/the-sourdough-framework#181] [Polls]
Hello,

I am wondering how people intend to "consume" the book... There are 3 versions ebook/pdf/website.  But I also made some decisions in book's pagination that might or might not be relevant if you read the pdf online as opposed to printing it.

@hendricius maybe you could also ask on discord what people think (you might need a github account to vote here..)

**Comment by fischerq** [comment]
I expect to do ~all of them, roughly for the following use cases:
* I've preordered the book. I'll occasionally pull it out of the bookshelf (to show to friends, or just to read a few paragraphs).
* I'll occasionally want to double check something (e.g. some ratio when mixing a dough). My smartphone will be closer than the book, so there is a good chance I'll use either the pdf or the website. It really depends on the ergonomics for finding a particular section. Probably this will be the website.
* I might pull out the book to read a larger part, e.g. during commute, while travelling, or when on vacation. I won't have the book with me, so I'll use whatever is more comfortable to read either on the phone or tablet (someone else might go for the ebook-reader). Ideal would be a .epub to load into Google Books, next best is probably the pdf.

**Comment by hendricius** [comment]
✅ done - I posted this on YouTube: https://www.youtube.com/post/Ugkx46-PmXZwc0fUoRcrv2POWvMxOgmTcPcR.

Personally - I think most people will at some point find the website through search engines. Then people might start to read digitally/print and ultimately some might be interested in buying the book. Let's see if we can get some more data on this.

The hardcover book will always be a bit behind the other versions. Making changes to it takes time and effort.

**Comment by cedounet** [comment]
Thanks guys.

Not trying to launch a debate on what format is best... Trying to work out what makes sense to do next.

- Shall we do a lower resolution/Smaller size ebook or is that not needed ?
- Right now we have a recto-verso PDF so it looks great if you print... but on screen that is a lot of empty pages which are a bit daft

- We can extend the makefile to support everything, of course, but at some point we should do what makes sense for most people and let other use the source.

**Reply by hendricius** [reply]
@cedounet the smaller resolution would be nice. Amazon has a 50mb limit that you can send to your kindle. This has caused some issues in the past.

Based on the vote - it seems most people opt for reading the PDF on their screen.

**Comment by cedounet** [comment]
Deal, I will amend the tex file accordingly.
